src/classes/RD2_ValidationService.cls

```diff
@@ -193,6 +193,11 @@ public with sharing class RD2_ValidationService {
             npe03__Recurring_Donation__c rd = rds[i];
             npe03__Recurring_Donation__c oldRd = oldRds[i];
 
+            if (elevateService.isElevateStatusChangeFromClosed(rd, oldRd)) {
```

Comment:
Is there a check anywhere to confirm that Elevate Integration is enabled? My concern is this validation will block customers that may choose to put a value in the CommitmentId even if they're not using Elevate Payments. I believe that in other places we do check for ElevatePayments enabled before enforcing validations. Though if that's not correct - and we ignore the elevate enabled status for other validations - then it can be ignored here. Basically just looking for consistency.